### PR TITLE
Write types pla 47

### PR DESCRIPTION
**Pull request by AnthonyArdronUW**

### CODE REVIEW

File: app/data-infra-bigquery-connector/main.go [hunk 1]
```diff
@@ -103,18 +112,57 @@ func processMsg(ctx context.Context, cat catalogv1.Catalog, fac connectors.Facto
...
+		logrus.Debugf("timestamp: %s", stamp)
+		tableId = fmt.Sprintf("%s_%s", tableId, stamp)
+		_, exists = bqcon.TableExists(datasetId, tableId)
+	default:
+		logrus.Errorf("Invalid write mode: %s", writeMode)
+		return fmt.Errorf("invalid write mode: %s", writeMode)
+	}
+
+	if exists != nil {
+		logrus.Debugf("creating table: %s", tableId)
```
Comment (paulosimao-ardanlabs) [inline app/data-infra-bigquery-connector/main.go:165]: guess this is dooubled w next line

File: app/data-infra-bigquery-connector/main.go [hunk 2]
```diff
@@ -103,18 +112,57 @@ func processMsg(ctx context.Context, cat catalogv1.Catalog, fac connectors.Facto
...
+		logrus.Infof("write mod was %s is now append", writeMode)
+		writeMode = catalogv1.WriteType_Append
+	}
+
+	logrus.Debugf("checking table exists: %s", tableId)
+	// An error should be returned if the table does not exist
+	_, exists := bqcon.TableExists(datasetId, tableId)
+
+	switch {
+	case writeMode == catalogv1.WriteType_Append:
```
Comment (paulosimao-ardanlabs) [inline app/data-infra-bigquery-connector/main.go:136]: a `switch writeMode` would be more simple., no need to test if mode == value in every case

**Review by paulosimao-ardanlabs — COMMENTED** [review]
Hi, please see comments inline

File: app/data-infra-bigquery-connector/main.go (hunk 2 above)
Comment (AnthonyArdronUW) [reply app/data-infra-bigquery-connector/main.go:136]: Good point

File: app/data-infra-bigquery-connector/main.go (hunk 1 above)
Comment (AnthonyArdronUW) [reply app/data-infra-bigquery-connector/main.go:165]: well spotted

**Review by paulosimao-ardanlabs — APPROVED** [review]

**Pull request merged**